_posts/2018-04-15-lazy-loading-images.md

```diff
@@ -0,0 +1,205 @@
+---
+title: 'Improve pagespeed: Lazy Loading Images'
+author: dimitri-steinel
+layout: post
+mainImage: ""
+mainImageAlt: ""
+--- 
+
+Don't want to read it all? [Here’s the code](#snippet)
+
+### First meaningful paint
+
+Have you heard of first meaningful paint? No? You should! The first meaningful paint is a very important parameter of you website. The longer a page needs to load the more the users lose interest and exit your website. So there is a very close connection between pagespeed and bounce rate.
+
+So what we want to achieve is, give the user a visible result as early as possible. There are many things you can do for speeding up the first meaningful paint. People talk about critical CSS, treeshaking JavaScript and brotli compression because gzip is not compressing it enough. They talk about avoid using libraries and help you to optimize your code to make it as small as possible. And they are right. All those things are a great improvement for pagespeed and pagespeed means more users will enjoy you website. But before we dive deeper in these topics, we start with optimizing images. There is nothing else which will improve your pagespeed more with that little effort.
```

Comment:
So what we want to achieve is to give the user a visible result as early as possible

Reply:
tree-shaking

Reply:
They talk about avoiding the use of libraries

Reply:
All those things are great improvements to page speed,  and a good page speed means more users will enjoy you website

Reply:
dive deeper into these topics